pom.xml

```diff
@@ -68,6 +68,7 @@
         <dep.casandra.version>4.14.0</dep.casandra.version>
         <dep.minio.version>7.1.4</dep.minio.version>
         <dep.iceberg.version>0.14.0</dep.iceberg.version>
+        <dep.aws-sdkv2.version>2.17.151</dep.aws-sdkv2.version>
```

Comment:
in https://github.com/trinodb/trino/pull/13294#discussion_r953760795 i meant also moving these code blocks

```
<dependency>
            <groupId>software.amazon.awssdk</groupId>
            <artifactId>s3</artifactId>
...
```

to root pom.xml's `<dependencyManagement>` section

Reply:
sorry about that, updated